testing/libneo4j-client/APKBUILD

```diff
@@ -21,17 +21,42 @@ build() {
             --prefix=/usr \
             || return 1
     make || return 1
-    make LIBS="-lfts" check || return 1
     make doc || return 1
 }
 
+check() {
+    cd "$_builddir"
+    make LIBS="-lfts" check || return 1
+}
+
 package() {
     cd "$_builddir"
     make DESTDIR="$pkgdir" install || return 1
     mkdir -p "$pkgdir"/usr/share/doc/libneo4j-client
     mv doc/html/* "$pkgdir"/usr/share/doc/libneo4j-client/
 }
 
+doc() {
+    depends="$depends_doc"
+    pkgdesc="$pkgdesc (documentation)"
+    install_if="docs $pkgname=$pkgver-r$pkgrel"
+
+    if [ "X$subpkgname" = "Xlibneo4j-client-doc" ]; then
```

Comment:
Oh my… please, this `x` non-sense in `[` is a workaround for a bug in one shell implementation that has been fixed several **decades** ago! It’s not needed anymore for very long time.

Reply:
Old habits, happy to remove.